packages/deprecation-crawler/src/lib/tasks/ensure-crawler-tsconfig.ts

```diff
@@ -0,0 +1,73 @@
+import { CrawlConfig } from "../models";
+import { prompt } from "enquirer";
+import { normalize } from "path";
+import { TSCONFIG_PATH } from "../constants";
+import { findTsConfigFiles } from "../config";
+import { readFileSync, writeFileSync } from "fs";
+import { format, resolveConfig } from "prettier";
+import { fileExists } from "@nrwl/workspace/src/utils/fileutils";
+
+export async function ensureTsConfigPath(config: CrawlConfig): Promise<CrawlConfig> {
+
+  const tsConfigFiles: string[] = findTsConfigFiles();
+  if (tsConfigFiles.length === 0) {
+    throw Error("We need a tsconfig file to crawl");
```

Comment:
Do we want to throw here? he could also create the tsconfig file.
Can be done later tho.

Reply:
Refactored to skip this step